src/main/resources/templates/clothing.html

```diff
@@ -101,7 +101,7 @@ <h2 class="container-selector">Size</h2>
                             <li class="cloth-size">
                                 <input class="form-check-input" type="checkbox" name="currency" id="usdCheckbox2"
                                        value="1">
-                                <label class="form-check-label" for="usdCheckbox">XS (5)</label>
+                                <label class="form-check-label" for="usdCheckbox">XS (1)</label>
```

Comment:
Eliminate (1), same repeated below many times

Reply:
will wait for changes

Reply:
ok im done for this one also